qupath-extension-openslide/src/main/java/qupath/lib/images/servers/openslide/jna/OpenSlideLoader.java

```diff
@@ -69,6 +69,19 @@ public static synchronized boolean tryToLoad(String... searchPath) throws Unsati
 
     private static synchronized OpenSlideJNA tryToLoadJnaInstance(String... searchPath) {
         String jnaPath = System.getProperty("jna.library.path", null);
```

Comment:
I struggled with the logic of this method; if the `jna.library.path` isn't used in the event that `searchPath` is empty, then I think better to delay requesting the property until closer to the time it is needed (i.e. after the `if` block).